### ISSUE TITLE
BVG API updated his auth.aid

### ISSUE THREAD

**Issue by FredNandrin** [public-transport/hafas-client#324]
Request are currently returning 
```
(...)
 "cInfo": {
        "code": "VD",
        "msg": "Damit du alle Features unserer Fahrplanauskunft und den Ticketkauf weiterhin problemlos nutzen kannst, ist es unerlässlich, dass du die neueste Version unserer App herunterlädst oder die App zeitnah aktualisierst."
    },
(...)```

**Comment by FredNandrin** [comment]
The fix is available ...
https://github.com/public-transport/hafas-client/pull/323

**Comment by derhuerst** [comment]
Fix published as `hafas-client@6.3.2`. 🎉
